elasticsearch/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM clearlinux:latest
+MAINTAINER qi.zheng@intel.com
+
+ARG swupd_args
+
+RUN swupd update $swupd_args \
+	&& swupd bundle-add elasticsearch $swupd_args \
+	&& rm -rf /var/lib/swupd/*
+
+RUN set -ex \
+	&& for path in \
+		/var/data/elasticsearch \
+		/var/log/elasticsearch \
+		/etc/elasticsearch \
+	; do \
+		mkdir -p "$path"; \
+		chown -R elasticsearch:elasticsearch "$path"; \
+	done
+
+COPY config /etc/elasticsearch
+COPY docker-entrypoint.sh /
+
+VOLUME /var/data/elasticsearch
+
+EXPOSE 9200 9300
+ENTRYPOINT ["/docker-entrypoint.sh"]
+CMD ["-Epath.conf=/etc/elasticsearch/"]
```

Comment:
Can you double check the command here? I would expect something like the [upstream CMD](https://github.com/docker-library/elasticsearch/blob/master/Dockerfile-debian.template#L66).

Reply:
I use this command "-Epath.conf=/etc/elasticsearch/" to set the default configuration path in /etc/ to respect the stateless design of Clear Linux.

Reply:
Using the upstream CMD implies the configuration path is in /usr/share/elasticsearch